js/flow/src/flow.ts

```diff
@@ -835,22 +835,24 @@ export function startFlowsServer(params?: {
   flows?: Flow<any, any, any>[];
   port?: number;
   cors?: CorsOptions;
+  pathPrefix?: string;
 }) {
   const port =
     params?.port || (process.env.PORT ? parseInt(process.env.PORT) : 0) || 3400;
+  const pathPrefix = params?.pathPrefix ? `/${params?.pathPrefix}` : '';
   const app = express();
   app.use(bodyParser.json());
   app.use(cors(params?.cors));
 
   const flows = params?.flows || createdFlows();
   logger.info(`Starting flows server on port ${port}`);
   flows.forEach((f) => {
-    logger.info(` - /${f.name}`);
+    logger.info(` - ${pathPrefix}/${f.name}`);
```

Comment:
```suggestion
    const flowPath = `${pathPrefix}${f.name}`;
    logger.info(` - ${flowPath}`);
```

Reply:
@pavelgj I think you meant to say ```const flowPath = `${pathPrefix}/${f.name}`;```? without the `/` it won't work properly

Reply:
The suggestion was that `params.pathPrefix` defaults to `/`, so `${pathPrefix}${f.name}` would be correct. The idea is that prefix doesn't necessarily need to have a `/` as separator. For example, if I want to a path like `/customPrefix_flowName` I'd just need to specify `/customPrefix_` as prefix. 

I'm thinking if it's required that the path starts with `/` then we can validate it....

Reply:
This feels very counter intuitive. I feel very rarely people would like to add a proper "prefix" to their paths rather than a parent route. having `/flows/joke` will be way more common than `/flows_joke`.

One thing we can do maybe is, have a proper "prefix" like what you're suggesting, but also through #198 we can enable devs to add parent routes like what i'm envisioning

Reply:
So there could be 4 options
- flow name only ex: `/joke`
- prefix + flow name ex: `/prefixjoke` (path has to start with `/` for express to work iirc)
- parent route + flow name ex: `/flows/joke` (through #198 )
- parent + prefix + flow name ex: `/flows/prefixjoke` (through #198 )

Reply:
Yes, and my thinking was that this is nicely covered by:

```js
const pathPrefix = params?.pathPrefix ?? '/';
if (pathPrefix.chatAt(0) !== '/') {
  pathPrefix = '/' + pathPrefix;
  // or throw new Error('pathPrefix must start with /')
}
const flowPath = `${pathPrefix}${f.name}`;
```

Reply:
@pavelgj made leading / default. It will now be `/flowName` or `/prefixflowName`